javascript/ql/src/semmle/javascript/frameworks/LodashUnderscore.qll

```diff
@@ -38,6 +38,319 @@ module LodashUnderscore {
    * In addition, the global variable `_` is assumed to refer to `lodash` or `underscore`.
    */
   DataFlow::SourceNode member(string name) { result.(Member).getName() = name }
+
+  /**
+   * Holds if `name` is the name of a member exported from the `lodash` package
+   * which has a corresponding `lodash.xxx` NPM package.
+   */
+  private predicate isLodashMember(string name) {
```

Comment:
Could we document how to update this? Ideally as the union of all exports from all lodash versions.
Maybe that documentation belongs outside of this file.

Reply:
I've added a comment mentioning how it was generated.